### PR TITLE
Add social media identifier class to SocialButton

### PR DESCRIPTION
**Pull request by marcilioreis**
#### What problem is this solving?

We currently can't customize the appearance of social media buttons without a way to identify them individually by name. 
I added classes to the button and icon to make this possible.

#### How to test it?

[Workspace](https://frnqa2--taafrl.myvtex.com/cerveja-duvel-666-gf-330ml/p!)

Just go to any product and check the included classes for customization

#### Screenshots or example usage:

Before:
image

After:
image

### CODE REVIEW

**Review by victorhmp — CHANGES_REQUESTED** [review]
The implementation LGTM!
But could you lint the file and update the CHANGELOG?

**Comment by marcilioreis** [comment]
> The implementation LGTM!
> But could you lint the file and update the CHANGELOG?



> The implementation LGTM!
> But could you lint the file and update the CHANGELOG?

Done!

File: react/components/Share/components/SocialButton.js [hunk 1]
```diff
@@ -50,29 +51,23 @@ export default class SocialButton extends Component {
...
-    const {
-      url,
-      message,
-      size,
-      socialEnum,
-      buttonClass,
-      iconClass,
-      imageUrl,
-    } = this.props
+    const { url, message, size, socialEnum, buttonClass, iconClass, imageUrl } =
```
Comment (igorbrasileiro) [inline react/components/Share/components/SocialButton.js:54]: Oh, that's interesting, our lint did this? Did you install the dependencies on the root folder?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.149.0] - 2021-07-20
```
Comment (igorbrasileiro) [inline CHANGELOG.md:10]: You don't need to bump the version manually, just remove this line and it will be ok. 
```suggestion
```

**Review by igorbrasileiro — CHANGES_REQUESTED** [review]
Hey, good job 👏 !
Left some comments, with small fixes

File: react/components/Share/components/SocialButton.js (hunk 1 above)
Comment (marcilioreis) [reply react/components/Share/components/SocialButton.js:54]: Yes, my vs code was complaing about prettier, so i've just installed

**Review by igorbrasileiro — APPROVED** [review]
Nice! You need rebase with master

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Added social media identifier class to SocialButton
+
```
Comment (victorhmp) [inline CHANGELOG.md:12]: This should resolve the merge conflicts pointed out by GitHub :)

```suggestion
## [Unreleased]
### Added
- Added social media identifier class to SocialButton

```

**Review by victorhmp — APPROVED** [review]
Awesome 👏🏻

**Comment by victorhmp** [comment]
@marcilioreis I've just noticed that the tests are not being executed for some reason... just to be sure, could you please run them locally in this branch to make sure everything is ok? Running `yarn test` should do it :)

**Comment by marcilioreis** [comment]
> @marcilioreis I've just noticed that the tests are not being executed for some reason... just to be sure, could you please run them locally in this branch to make sure everything is ok? Running `yarn test` should do it :)

Hi, 

I just run the test as you asked and everything went well

image

**Pull request merged**